src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -740,6 +740,66 @@
   <data name="OpenCWDCommandKey" xml:space="preserve">
     <value>Open current working directory</value>
   </data>
+  <data name="CloseTab" xml:space="preserve">
+    <value>Close tab</value>
+  </data>
+  <data name="SendInput" xml:space="preserve">
+    <value>Send input</value>

```

Comment:
hmm we already have some of these and they use different capitalization (`SendInputCommandKey` for example)

Is it possible for you to go through the ones with matching `CommandKey` entries and make the new ones consistent?

Reply:
lol. sure, let's do it. good catch